dashboard/src/components/AppView/AppSecrets/AppSecrets.scss

```diff
@@ -1,4 +1,6 @@
 .app-secrets-content {
-  margin-top: 1.2rem;
-  margin-right: 1.2rem;
+  margin-top: 1rem;
+  padding-right: 15px;
+  max-height: 15rem;
+  overflow-y: auto;
```

Comment:
this was necessary (and related to https://github.com/kubeapps/kubeapps/issues/2462) when the app has too many secrets (e.g. harbor):

![Screenshot from 2021-03-17 12-39-05](https://user-images.githubusercontent.com/4025665/111462019-32718080-871e-11eb-9064-14d49e07be2d.png)